engine/access/rest/routes/account_keys_test.go

```diff
@@ -0,0 +1,134 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	mocktestify "github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/access/mock"
+	"github.com/onflow/flow-go/model/flow"
+
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// TestGetAccountKeyByID tests local getAccount request.
+//
+// Runs the following tests:
+// 1. Get key by address and ID at latest finalized block
+// 2. Get missing key by address and ID at latest finalized block
+// 3. Get invalid account.
+func TestGetAccountKeyByID(t *testing.T) {
+	backend := &mock.API{}
+
+	t.Run("get key by address and ID at latest finalized block", func(t *testing.T) {
+		account := accountFixture(t)
+		var height uint64 = 100
+		block := unittest.BlockHeaderFixture(unittest.WithHeaderHeight(height))
+
+		req := getAccountKeyByIDRequest(t, account, "0")
+
+		backend.Mock.
+			On("GetLatestBlockHeader", mocktestify.Anything, false).
+			Return(block, flow.BlockStatusSealed, nil)
+
+		backend.Mock.
+			On("GetAccountAtBlockHeight", mocktestify.Anything, account.Address, height).
+			Return(account, nil)
+
+		expected := expectedAccountKeyResponse(account)
+
+		assertOKResponse(t, req, expected, backend)
+		mocktestify.AssertExpectationsForObjects(t, backend)
+	})
+
+	t.Run("get missing key by address and ID at latest finalized block", func(t *testing.T) {
```

Comment:
can you also add a test for when the address is missing?

Reply:
Added more test cases in https://github.com/onflow/flow-go/pull/4583/commits/29402e349eb8e6b86a5da8f4e530c2a817e09dea